packages/runtime/src/util/types.ts

```diff
@@ -0,0 +1,19 @@
+import type { Widget } from "framework";
+
+export interface EnsembleWidgetProps {
+  id?: string;
+  [key: string]: unknown;
+}
+
+export type FlexboxProps = {
```

Comment:
Props should be colocated with the implementation

Reply:
This is being used by Row/Column/Flex hence it's outside.

Reply:
Got it, I'll make a Layout folder and put colocate all the containers and their typings in there.